Libraries/Blob/RCTBlobManager.mm

```diff
@@ -176,6 +176,7 @@ - (void)remove:(NSString *)blobId
 
 RCT_EXPORT_METHOD(createFromParts:(NSArray<NSDictionary<NSString *, id> *> *)parts withId:(NSString *)blobId)
 {
+  std::lock_guard<std::recursive_mutex> lock(_blobsMutex);
```

Comment:
Why do we need a lock here? We only change instance state in `store` which already locks.